components/calendar/src/astronomy.rs

```diff
@@ -480,7 +493,6 @@ impl Astronomical {
     }
 
     /// The longitude of the Sun at a given Moment in degrees
-    #[allow(dead_code)] // TODO: Remove dead_code tag after use
     pub(crate) fn solar_longitude(moment: Moment) -> f64 {
```

Comment:
Suggestion: does this function always return an integer between 0 and 359? Then the return type should be `usize` or even `u16`